content/docs/command-reference/checkout.md

```diff
@@ -118,6 +118,14 @@ $ git clone https://github.com/iterative/example-get-started
 $ cd example-get-started
 ```
 
+This project comes with a predefined HTTP
+[remote storage](/doc/command-reference/remote). We use `dvc fetch` to get data
+from remote storage:
+
+```dvc
```

Comment:
```suggestion
```
Just add the fetch command to the initial code block (removing all the lines above) so this is brief.

Reply:
Just please add a sentence after the code block explaining why we're fetching -aT here.